.github/workflows/publish_reports.yml

```diff
@@ -2,7 +2,7 @@ name: Publish reports
 
 on:
   workflow_run:
-    workflows: ["Build And Test"]
+    workflows: ["Build and Test main tests on Pull Requests", "Build and Test Template tests on Pull Requests"]
```

Comment:
This seems like an odd name for the workflow. Are we testing the tests? Or testing the PR?

Reply:
we are testing the product when a pull request is created